rudder-sdk-core/src/main/java/com/rudderlabs/android/sdk/core/Constants.java

```diff
@@ -14,4 +16,6 @@
     // if events are registered and flushQueueSize is not reached
     // events will be flushed to server after sleepTimeOut seconds
     static final int SLEEP_TIMEOUT = 10;
+    // config-plane url to get the config for the writeKey
+    static final String CONFIG_PLANE_URL = "https://api.rudderlabs.com/";
```

Comment:
Generally, a good practice to have the URL without "/" and add it later when needed.

That makes it more readable when appending the endpoint to the URL.
Eg. configUrl becomes  CONFIG_PLANE_URL + "/sourceConfig"